lib/keybindings-panel.coffee

```diff
@@ -73,10 +73,13 @@ class KeybindingsPanel extends ScrollView
     @keybindingRows.empty()
     for keyBinding in keyBindings
       {selector, keystrokes, command, source} = keyBinding
+      source = KeybindingsPanel.determineSource(source)
```

Comment:
The original source is something like:
```
/Users/octref/.atom/packages/vim-mode-plus/keymaps/vim-mode-plus.cson
/Applications/Atom.app/Contents/Resources/app.asar/keymaps/darwin.json
```
This makes them
```
Vim Mode Plus
Core
```

Which is more what user would search for, I think.